### ISSUE TITLE
Removal of PermissionName broke downstream standards

### ISSUE THREAD

**Issue by annevk** [w3c/permissions#366]
#353 broke the Storage Standard and I suspect other standards are impacted as well. What's the migration story?

https://github.com/whatwg/storage/pull/135 is blocked on this. Should I just remove the reference?

**Comment by marcoscaceres** [comment]
yeah, sorry about that... my plan was to fix them as we build the registry as I wanted to double check each one meets the criteria, etc.

**Comment by annevk** [comment]
What should I do meanwhile, just remove the reference I guess?

**Comment by marcoscaceres** [comment]
Sent https://github.com/whatwg/storage/pull/136 - but leaving this open so I can track down the rest.
